### PR TITLE
Update codebase to python 3.7

### PR DESCRIPTION
**Pull request by initial1ze**
Let me know if there is an issue or something needs to be improved.
Thank you.

### CODE REVIEW

File: selene/support/shared/config.py [hunk 1]
```diff
@@ -386,11 +386,11 @@ def type_by_js(self, value: bool):
         self._type_by_js = value
 
     @Config.window_width.setter
-    def window_width(self, value: Optional[int]):
+    def window_width(self, value: int | None):
         self._window_width = value
 
     @Config.window_height.setter
-    def window_height(self, value: Optional[int]):
+    def window_height(self, value: int | None):
```
Comment (aleksandr-kotlyar) [inline selene/support/shared/config.py:393]: This is a python 3.10 feature https://www.python.org/dev/peps/pep-0604 
It shouldn't be here. The task is to up to 3.7 python, not 3.10.

File: selene/core/condition.py [hunk 2]
```diff
@@ -44,7 +44,7 @@ def fn(entity):
     @classmethod
     def by_or(cls, *conditions):
         def fn(entity):
-            errors: List[Exception] = []
+            errors: list[Exception] = []
```
Comment (aleksandr-kotlyar) [inline selene/core/condition.py:47]: It's not python 3.7 - it's python 3.9 feature https://www.python.org/dev/peps/pep-0585/
It shouldn't be here for now.

**Review by aleksandr-kotlyar — CHANGES_REQUESTED** [review]
@initial1ze thank you for your time to make this PR.
You need to rollback some changes from 3.10 and 3.9 python before we could approve your PR.

**Comment by initial1ze** [comment]
Sorry for the trouble, i will make the required changes.

**Comment by aleksandr-kotlyar** [comment]
That was an attempt to resolve #288

**Pull request closed without merging**